bucket/zeplin.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
```

Comment:
Use something like: `Collaborative tool for handoff designs with specs, ...`

Reply:
Fixed!